app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: :AuthorId
+  has_many :posts, foreign_key: :AuthorId
+  has_many :likes, foreign_key: :AuthorId
```

Comment:
- [ **OPTIONAL** ] Good job :+1:  to use the correct associations for your the User modal. But I think it would be better if you use the camel case notation on the `AuthorId` key every where you use it in your App. This for rails good practices